exporters/zipkin/zipkin.go

```diff
@@ -65,6 +69,11 @@ func (fn optionFunc) apply(cfg config) config {
 
 // WithLogger configures the exporter to use the passed logger.
 func WithLogger(logger *log.Logger) Option {
```

Comment:
We should have some comments that `WithLogger` and `WithLogr` will overwrite each other.